packages/experiment-tag/src/util.ts

```diff
@@ -79,9 +79,7 @@ export const concatenateQueryParamsOf = (
   const urlObj = new URL(redirectUrl);
 
   globalUrlObj.searchParams.forEach((value, key) => {
-    if (!urlObj.searchParams.has(key)) {
-      urlObj.searchParams.set(key, value);
-    }
+    urlObj.searchParams.append(key, value);
```

Comment:
Would like to double check if this change would break anything related to redirects? @tyiuhc

Reply:
this should be fine 👍🏻

Reply:
Oh yes, going to fix this. 
Changing the behavior to be everything from redirect url plus everything in original url that doesn't conflict with redirect url.